lib/parsers/txt.mjs

```diff
@@ -158,7 +159,9 @@ export async function parse (rawText, filename) {
       nonReferenceSectionRfc: [],
       referenceSectionRfc: [],
       nonReferenceSectionDraftReferences: [],
-      referenceSectionDraftReferences: []
+      referenceSectionDraftReferences: [],
+      bracketedRfcReferences: [],
+      bracketedRfcDraftReferences: []
```

Comment:
What is a draft RFC? Draft - is a non-submitted document that may become an RFC. And we already collect bracketed draft references from reference and other text section.